packages/zoe/test/unitTests/contracts/escrowToVote.js

```diff
@@ -0,0 +1,118 @@
+/* global harden */
+// @ts-check
+
+// Eventually will be importable from '@agoric/zoe-contract-support'
+import { assert } from '@agoric/assert';
+import makeStore from '@agoric/store';
+import { makeZoeHelpers } from '../../../src/contractSupport';
+
+/**
+ * Implement coin voting. Give a voting capability when a payment is
+ * escrowed.
+ *
+ * @typedef {import('../../../src/zoe').ContractFacet} ContractFacet
+ * @typedef {import('@agoric/ERTP').Amount} Amount
+ * @param {ContractFacet} zcf
+ */
+const makeContract = zcf => {
+  const { assertKeywords, checkHook } = makeZoeHelpers(zcf);
+  assertKeywords(harden(['Assets']));
```

Comment:
Where do you validate that Assets is `'nat'` fungible? I expected you to assume it below, but see comment on line 92.

Reply:
I changed it to use amountMath.add so that both fungible and non-fungible could work

Reply:
I can't think of any plausible uses of voting where the votes aren't aggregated by summing or counting.  (And the easiest way to represent counting is to make sure all the assets have an extent of 1.) Sometimes there are distinct votes by class (class A shares vs. class B shares, e.g.). 

If we don't have a plausible story of how we'd support something else, I think this would make more sense as an example if it just used nat math.